### PR TITLE
MF-205 - Add permission to admin to list all groups

### LINKED ISSUE #205: Add admin permission to list groups.
Currently only user can list groups. Add permission to the administrator so he can  list them too.

### PR DESCRIPTION
**Pull request by zzokki81**
Added permission to admin to list all groups.

Resolves #205

### CODE REVIEW

File: things/postgres/groups.go [hunk 1]
```diff
@@ -523,11 +463,90 @@ func (gr groupRepository) UnassignMember(ctx context.Context, groupID string, id
...
 
-type dbMember struct {
-	MemberID  string    `db:"member_id"`
-	GroupID   string    `db:"group_id"`
-	CreatedAt time.Time `db:"created_at"`
-	UpdatedAt time.Time `db:"updated_at"`
+func (gr groupRepository) retrieve(ctx context.Context, ownerID string, pm things.PageMetadata) (things.GroupPage, error) {
+	var ownq string
+	if ownerID != "" {
+		ownq = "owner_id = :owner_id"
```
Comment (manuio) [inline things/postgres/groups.go:467]: Can we use `getOwnerQuery` here?

**Review by manuio — CHANGES_REQUESTED** [review]

File: things/postgres/groups.go (hunk 1 above)
Comment (zzokki81) [reply things/postgres/groups.go:467]: I don't think so.  In group SQL table we have `owner_id` and in channel SQL tale `owner`

Comment (manuio) [reply things/postgres/groups.go:467]: Right!

**Review by manuio — APPROVED** [review]
LGTM

**Review by manuio — APPROVED** [review]
LGTM

**Pull request merged**